.vscode/launch.json

```diff
@@ -0,0 +1,45 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "name": "Extension",
+        "type": "extensionHost",
+        "request": "launch",
+        "runtimeExecutable": "${execPath}",
+        "args": [
+            "--extensionDevelopmentPath=${workspaceFolder}"
+        ],
+        "env": {
+            "LSEXTENSIONS_PATH":  "",
+            "SERVER_DEBUG": "false",
+            "COMPOSERDEBUG": "false",
+            "COMPOSERDEV_HOST": "http://localhost:9000"
+        },
+        "outFiles": [
+            "${workspaceFolder}/dist/**/*.js"
+        ],
+        "preLaunchTask": "npm: watch"
+    },
+    {
+        "name": "Extension Tests",
+        "type": "extensionHost",
+        "request": "launch",
+        "runtimeExecutable": "${execPath}",
+        "env": {
+            "LSEXTENSIONS_PATH":  "",
+            "SERVER_DEBUG": "false",
+            "COMPOSERDEBUG": "false"
```

Comment:
Same as previous

Reply:
Removing only the first option, as discussed.